EcalTiming/python/ecalLocalRecoSequenceAlCaStream_cff.py

```diff
@@ -0,0 +1,11 @@
+from RecoLocalCalo.Configuration.ecalLocalRecoSequence_cff import *
```

Comment:
I think that ultimately we will switch to the pi0 stream, and since the two formats are the same, apart the digi collection name that we change at the level of cfg, the best is to have the reconstruction sequence which is decoupled from what it is done in the phi-symmetry or pi0 packages. With that, we don't need further packages apart EcalTiming.